.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
```

Comment:
Thanks for adding Travis support, I'll enable this repo for it. Can you put this change up as a separate PR?